content/comments-in-swift.md

```diff
@@ -0,0 +1,44 @@
+<!--
+---
+Subjects: # Please first look to use Subjects that are already listed at author.codecademy.com/browse/categories
```

Comment:
I'm not seeing these two Subjects in the author link. Did the terms there not seem adequate? (I think we'll probably need to create an expanded list for Codepedia, so if the answer is yes, that's really helpful feedback)